resources/quality/ultimaker_s5/um_s5_aa0.25_um-pla_0.1mm.inst.cfg

```
@@ -0,0 +1,32 @@
+[general]
+definition = ultimaker_s5
+name = Fine
+version = 4
+
+[metadata]
+material = ultimaker_pla
+quality_type = normal
+setting_version = 22
+type = quality
+variant = AA 0.25
+weight = 0
+
+[values]
```

Need to enable the acceleration since it is now disabled by default
```suggestion
[values]
_plugin__curaenginegradualflow__0_1_0__gradual_flow_enabled = True
```